src/less/site.less

```diff
@@ -193,6 +228,46 @@ input {
     right: 0;
     top: 0;
   }
+
+  .map-diaspora {
+    position: absolute;
+    right: 0;
+    top: 15px;
+
+    @media (min-width: 480px) {
+      top: 60px;
+    }
+
+    @media (min-width: 992px) {
+      right: 60px;
+    }
+  }
+
+  #map-diaspora {
+    height: 30px;
+    margin: 0 auto 10px;
+    width: 30px;
+
+    @media (min-width: 480px) and (max-width: 767px) {
+      height: 60px;
+      width: 60px;
+    }
+
+    @media (min-width: 768px) and (max-width: 991px) {
+      height: 90px;
+      width: 90px;
+    }
+
+    @media (min-width: 992px) and (max-width: 1199px) {
+      height: 60px;
+      width: 60px;
+    }
+
+    @media (min-width: 1200px) {
+      height: 90px;
+      width: 90px;
+    }
+  }
```

Comment:
Revert all this. This is not the case anymore, the diaspora map doesn't even exist.